addons/calendar/static/tests/activity_widget_tests.js

```diff
@@ -14,17 +14,18 @@ QUnit.test("list activity widget: reschedule button in dropdown", async (assert)
         icon: "fa-calendar",
         name: "Meeting",
     });
+    const tomorrow = luxon.DateTime.now().plus({days:1}).toFormat("yyyy-MM-dd");
```

Comment:
I think a const { DateTime } = luxon at the beginning of the file would ge good here (same thing elsewhere)